pypit/arproc.py

```diff
@@ -504,10 +508,12 @@ def flatnorm(slf, det, msflat, maskval=-999999.9, overpix=6, plotdesc=""):
     from pypit import arcyutils
     from pypit import arcyextract
     from pypit import arcyproc
+    dnum = 'det{0:02d}'.format(det)
```

Comment:
Yep, very good idea - I've now implemented this throughout the code. However, I can't imagine there being more than 99 detectors in a mosaic   :-O